CONTRIBUTING.md

```diff
@@ -0,0 +1,221 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+
+You can contribute in one of the following ways:
+
+1. Submit a bug report or a feature
```

Comment:
These items should be in an unordered list 
* Submit a bug report
* Submit a pull request
* Contribute to
* Reach us

An ordered list (numbers) is for procedures.

Reply:
fixed